packages/grid/test/selection-column-lazy-import-lit.test.js

```diff
@@ -1,13 +0,0 @@
-import { fixtureSync } from '@vaadin/testing-helpers/dist/fixture.js';
-
-it('should not throw when grid with items imported lazily after selection column', async () => {
```

Comment:
Looks like this file was renamed to `selection-column-lazy-import.js` (without `.test.js`) so the corresponding Lit test is also not generated.

Reply:
Good catch, fixed.